cfme/tests/services/test_rest_services.py

```diff
@@ -2105,3 +2107,184 @@ def test_deny_service_ordering_via_api(
     with pytest.raises(APIException):
         template.action.order()
     assert_response(appliance, http_status=400)
+
+
+@pytest.fixture(scope="module")
+def set_run_automate_method(appliance):
+    reset_setting = appliance.advanced_settings["product"][
```

Comment:
optional:
In state of specific with parsing i would make copy of current and restore at tear-down. but its good :+1:

Reply:
I am restoring the original state at tear-down. I don't understand what you're suggesting.

Reply:
btw its optional I was saying copy of `appliance.advanced_settings`

Reply:
Oh. I usually restore the appliance in the original state after the test is done.